tests/dummy/app/pods/components/frost-create-form-with-scroll/template.hbs

```diff
@@ -1,27 +0,0 @@
-{{#frost-modal-input title='Test title' subtitle='Subtitle'}}
```

Comment:
just temporarily removing this so I remember to update this at the end with the working version